main.go

```diff
@@ -200,6 +200,12 @@ func main() {
 			DefaultText: "false, ie enable ActionCache dependency checks",
 			EnvVars:     []string{"BAZEL_REMOTE_DISABLE_GRPS_AC_DEPS_CHECK"},
 		},
+		&cli.BoolFlag{
+			Name:        "enable_endpoint_metrics",
```

Comment:
Please also update the `--help` text in README.md.

Reply:
done !